docs/release-notes/master.adoc

```diff
@@ -6,7 +6,7 @@ include::topics/templates/document-attributes.adoc[]
 :context: release-notes
 :release-notes:
 
-= {ReleaseNotesName}
+= Release Notes
```

Comment:
Missing id. All assemblies and modules require the id. Only difference is that module IDs have `_{context}` at the end.